client/components/data/media-list-data/index.jsx

```diff
@@ -3,15 +3,17 @@
  */
 var React = require( 'react' ),
 	assign = require( 'lodash/assign' ),
-	isEqual = require( 'lodash/isEqual' );
+	isEqual = require( 'lodash/isEqual' ),
+	connect = require( 'react-redux' ).connect;
```

Comment:
Let's not bother with consistency. Avoid new `require` and use ES2015 import instead.